modules/standard/List.chpl

```diff
@@ -163,7 +160,6 @@ module List {
       Used in new expressions.
 
       :arg other: The list to initialize from.
-      :type other: `list(?t)`
```

Comment:
This raises the question of how, in general, should generic types be documented?